app/controllers/websocket_authentication_controller.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
```

Comment:
I like having a separate controller a lot better than having a public method in the `job_connection_controller`. Nice work on separation of concerns! 💯